aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -152,7 +152,7 @@ def get_builder_from_protocol(
             an initial guess for the magnetic moment is automatically set in case neither is provided.
```

Comment:
So now the `overrides` takes precedence over the `initial_magnetic_moments`, so we need to update the doc